### PR TITLE
Update DynamicBudgetTable to fill more of the view when viewing fewer months

### PR DESCRIPTION
**Pull request by jarneson33**
Hello, new to actual and loving the project! That said I was taken aback by the budgets view when only viewing 1-2 months on a high resolution panel. So I am opening this pull request to see if this is something the maintainers would be open to or if this fixed width is a mandatory design choice.

EDIT:
Changed from allowing 100% span to just upping the original logic to allow the table to maintain original design but be less floaty island.

Thank you.

Before

image
image

After

image
image

### CODE REVIEW

**Comment by Crazypkr1099** [comment]
Not a fan of this. Having it stretch across looks extremely weird. I'd be open to 50% instead of 100% 🤷

**Comment by Teprifer** [comment]
While there may be scope to allow the table to be a bit wider, the current images shown are too extreme. The data is divorced from the categories in single month and even two month views as the eye has to travserse too far from one side to the other. 

Maybe put up a few set with different percentages for people to review and give feedback?

**Comment by jarneson33** [comment]
@Crazypkr1099 @Teprifer yep I hear you guys, I agree too extreme in the other direction. I have pushed a changed to just allow the table to be a bit larger instead of doing any percentages. The table maintains original design but fills the space nicer. My main use case is doing two months side by side and this change allows  two months to nicely fill the space and for the table to not look distorted from doing a full span when looking at 1-2 months.

Please review updated screenshots or the PR branch demo and let me know your thoughts.

Thank you.

**Comment by Teprifer** [comment]
Seems reasonable, keen to hear other's thoughts.

**Comment by MatissJanis** [comment]
👋 +1 to what has already been said before - I'm not particularly excited about stretching out the default width.

HOWEVER: what if we made the width draggable? The left side-nav recently got a feature where you can drag its border to increase its width. https://github.com/actualbudget/actual/pull/2993

What if we set the same logic for the budget table too? That would keep the existing dimensions, but allow users to expand it if they so wish.

File: upcoming-release-notes/2933.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+category: Features
```
Comment (MatissJanis) [inline upcoming-release-notes/2933.md:2]:
```suggestion
category: Enhancements
```

**Comment by Jonathan-Fang** [comment]
> HOWEVER: what if we made the width draggable? The left side-nav recently got a feature where you can drag its border to increase its width. #2993
> 
> What if we set the same logic for the budget table too? That would keep the existing dimensions, but allow users to expand it if they so wish.

I think this is a good idea; my workaround is just making all the text bigger, like 110% zoomed in. It's a workaround but at some point it would be nice to have a native zoom-in feature, or maybe it's because of my 1920x1080 monitor ratio that the default just looks so empty with whitespace.

**Pull request closed without merging**